frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -99,43 +84,40 @@ export default function Dashboard(props: Props) {
 	
 	const loggedInDropdown = () => {
 		return <Dropdown button={ <div style={ {
-			aspectRatio: "1",
-			display: "grid",
-			placeItems: "center",
 			userSelect: "none",
 			cursor: "pointer",
 			borderRadius: "50%",
-			boxShadow: "var(--box-shadow)"
+			outlineOffset: "var(--outline-width)"
 		} }
 									   tabIndex={ 0 }>
-			<Avatar userId={ session?.identity?.id }/>
+			<Avatar userName={ props.profile?.name }/>
 		</div> }
 						 items={ [
 							 {
 								 icon: "fi fi-rr-user",
-								 label: "Profile",
-								 shortcut: <span className={ "badge" }>Pro</span>,
+								 label: props.profile?.name ?? "Profile",
```

Comment:
Due to i18n: Extract hardcoded text into translation file, allowing for easier translation and maintenance of multilingual requirement.

Reply:
This is a fallback and will only be visible if there is an error in the backend.